testkit/javadsl/src/main/scala/com/lightbend/lagom/javadsl/testkit/PersistentEntityTestDriver.scala

```diff
@@ -235,7 +235,7 @@ class PersistentEntityTestDriver[C, E, S](system: ActorSystem, entity: Persisten
   private def checkSerialization(obj: Any): Option[Issue] = {
     val obj1 = obj.asInstanceOf[AnyRef]
     // check that it is configured
-    Try(serialization.serializerFor(obj.getClass)) match {
+    Try(serialization.findSerializerFor(obj1)) match {
```

Comment:
This is the fix, everything else is just test code (which I wrote test-first to be sure it failed without this change).

The change is just to use a method in Akka's `Serialization` class that already handles `null` safely. This method also exists in Akka 2.4, so this is safe to backport.

I also confirmed that Akka Persistence Cassandra uses `findSerializerFor` when actually making snapshots, and Akka Persistence JDBC uses it indirectly by calling `Serialization.serialize`.